README.md

```diff
@@ -1 +1,205 @@
+
+
+## Overview
+
+This repository contains the first version of adapting the [new generation][blog-new-generation] of [Snowplow's BiqQuery web data model v1][bq-web-model-dir] to be run with [Dataform][dataform]. As such and in order to keep an exact correspondence, in v0.1.0 we intentionally and explicitly enforced the incrementalization logic and modular structure of the parent model.
+
+That way the Snowplow Dataform's web data model inherits as key features:
+
+ - Extensible modular structure
+ - Metadata log tables
+ - Assertions to test your data quality
+ - Feature flags
+
+You can read more about the structure, logic and technical architecture of the Snowplow web data model:
+ - in the [README][readme-web-v1] of the official data-models repository
+ - in the Snowplow [blog][blog-bigquery]
+ - in our [docs][docs-datamodeling]
+
+As an overview the standard web data model runs 4 core modules:
+
+```
+.
+├── dataform.json
+├── definitions
+│   ├── assertions
+│   ├── standard
+│   │   ├── 00_setup
+│   │   ├── 01_base
+│   │   ├── 02_page_views
+│   │   ├── 03_sessions
+│   │   └── 04_users
+│   └── utils
+│       └── declarations.js
+├── environments.json
+├── includes
+│   └── sp.js
+├── package.json
+└── package-lock.json
+
+```
+
+1. **01_base**
+   The Base module applies incremental logic to the atomic data, and produces deduplicated tables for the Page Views module.
+
+2. **02_page_views**
+   The page views module runs the standard web page views model. It takes the `events_staged` table - produced by the Base module - as an input (and also queries atomic data). Its persistent output is the `derived.page_views` table.
+
+3. **03_sessions**
+   The sessions module runs the standard web sessions model - it takes the `page_views_staged` table - produced by the Page Views module - as an input. Its persistent output is the `derived.sessions` table.
+
+4. **04_users**
+   The users module runs the standard web users model - it takes the `sessions_userid_manifest_staged` table - produced by the Sessions module - as an input. Its persistent output is the `derived.users` table.
+
+
+## Configuration
+
+To configure your data model you only need to check the `sp.js` file in your `includes` directory:
+
+### Parameters
+
+1. **Schema names**
+   - `input_schema`: name of atomic dataset
+   - `scratch_schema`: name of scratch dataset
+   - `output_schema`: name of derived dataset
+
+   ```
+   const input_schema = "rt_pipeline_prod1";
+   const scratch_schema = "scratch";
+   const output_schema = "derived";
+   ```
+
+2. **Dataform specific**
+   - `model_disabled`: (Default: `false`). Provides a way to disable the model. It affects the 01-main and the 99-complete steps.
+   - `assertions_disabled`: (Default: `false`). Whether assertions are run along the model.
+   - `destroy_disabled`: (Default: `true`). Provides a way to drop all tables produced by the model for a full rerun. _Note_: this does not affect the `datamodel_metadata` table.
```

Comment:
Like in the `XX_destroy` playbooks of the parent model (for example [XX-destroy-base.yml.tmpl](https://github.com/snowplow/data-models/blob/master/web/v1/bigquery/sql-runner/playbooks/standard/01-base/XX-destroy-base.yml.tmpl), we run both the `99-complete` and the `XX-destroy` sql steps. So they do depend on the complete steps. However, it is not necessary to run the main steps:
Whether the main steps run depends on the `model_disabled`. If this is set to true, the main steps do not run. 
So, if we set `model_disabled` to true and `destroy_disabled` to false, it will only run the `99-complete` and `XX-destroy` steps, for all modules.
Only if both are set to false, the model will run the entire model and then destroy all output.

Reply:
Ok cool that works! Thanks for clarifying.